scripts/cbmc-viewer/cbmc-quiet-xml

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/python -u
+
+import re
+import sys
+
+REGEXP = [
+    (r'replacing function pointer by .* possible targets.*'),
+    (r'^Unwinding loop .* iteration .* \(.* max\)'),
+    (r'^Not unwinding loop .* iteration .* \(.* max\)'),
+    r'^\[.*\] .*: SUCCESS',
+    r'^\[.*\] .*: FAILURE',
+    r'Covered block .* \(lines .*\)'
+]
+
+def match_string (string):
+    for regexp in REGEXP:
+        if re.match(regexp,string):
+            return True
+    return False
```

Comment:
this file seems to have a bunch of copy&pasted code from the other cbmc-quiet

Reply:
This is an old script has has been removed.